README.md

```diff
@@ -1,7 +1,7 @@
 # Azure Pipelines YAML
 
 YAML templates, samples, and community interaction for [Azure Pipelines](https://docs.microsoft.com/azure/devops/pipelines/).
-While this doesn't replace the docs, our fantastic [support team](https://azure.microsoft.com/support/devops/), or our [UserVoice](https://visualstudio.uservoice.com/forums/330519-visual-studio-team-services), feel free to file suggestions and issues here if they're related to YAML pipelines.
+While this doesn't replace the docs, our fantastic [support team](https://azure.microsoft.com/support/devops/), or our [UserVoice]((https://developercommunity.visualstudio.com/spaces/21/index.html), feel free to file suggestions and issues here if they're related to YAML pipelines.
```

Comment:
Maybe replace UserVoice with Developer Community, since the former is a product that isn't being used